src/use-replicant.ts

```diff
@@ -3,7 +3,7 @@ import { klona as clone } from "klona/json";
 
 type JsonValue = boolean | number | string | null;
 
-type Json = JsonValue | JsonValue[] | { [key: string]: Json };
+type Json = JsonValue | JsonValue[] | { [key: string]: Json } | Json[];
```

Comment:
May I suggest simply importing and using the [Jsonifiable type from the type-fest package](https://github.com/sindresorhus/type-fest/blob/main/source/jsonifiable.d.ts)? That will cover other edge cases for you like objects with a toJson method, and read-only arrays which both would be rejected by this type.

Reply:
Tested locally and the Jsonifiable type works. @Hoishin thoughts?

Reply:
Adding package just for this simple type is overkill for me. Edge cases you mentioned don't happen in this case.